.github/workflows/vib.yaml

```diff
@@ -0,0 +1,44 @@
+name: 'vib'
+on: # rebuild any PRs and main branch changes
+  pull_request:
+    branches:
+      - stable
+    types:
+      - opened
+      - synchronize
+      - reopened
```

Comment:
I **think** you don't need these anymore, they are the default types for [`pull_request`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request)